pallets/claim/README.md

```diff
@@ -0,0 +1,3 @@
+### Claim module
+
+Forked off from `polkadot/runtime/common` module `claim` at commit: [177b03958c766fe053f28424ee6f6748644bb794](https://github.com/paritytech/polkadot-sdk/tree/177b03958c766fe053f28424ee6f6748644bb794/polkadot/runtime/common/src/claims)
```

Comment:
It is a release commit? Can we use stable2509-2 version or any other release version? Probably the code is intact for a long time, can we just change the commit hash here?